docker/README.md

```diff
@@ -28,7 +38,8 @@ It's also possible to use `tests` and `factory` from within the `openqa_data` co
 to ditch `openqa_data` container altogether (so you have only `webui` and `worker` containers and data is loaded and saved completely into your host
 system). If this is what you want, refer to [Keeping all data in the Data Volume Container] and [Keeping all data on the host system] sections respectively.
 
-## Update firewall rules
+**Update firewall rules (Fedora)**
```

Comment:
Since we are describing Fedora and openSUSE images, it is no more a header, because there are text after that which is not inside this "section of text". It become an optional action to do if you are using Fedora images and then it is only a marker. Just maintained bold for eye-search.

But I can make it an own section and the text after that too if you prefer.